protocols/kad/src/kbucket/key.rs

```diff
@@ -92,10 +91,10 @@ impl<T> Into<KeyBytes> for Key<T> {
     }
 }
 
-impl From<Multihash> for Key<Multihash> {
-    fn from(m: Multihash) -> Self {
-        Key::new(m)
-    }
+impl From<Vec<u8>> for Key<Vec<u8>> {
```

Comment:
Do we need this instance?

Reply:
That is my workaround to convert a `Multihash` to a `Key` (as `Key<Multihash> is missing). You then call `Key::from(my_multihash.to_bytes()`.